packages/paste-website/stories/Pricing.stories.tsx

```diff
@@ -0,0 +1,378 @@
+          price="$99"
+          packageList={[
+            "Up to 2,100 successful verifications",
+            "4.76% discount compared to pay-as-you-go",
+            "Cancel anytime",
+          ]}
+          isRecommended
+        />
+      </VisualPickerRadioGroup>
+    </Box>
+  );
+};
+
+export const TableExample = (): JSX.Element => {
+  return (
+    <Table>
+      <THead>
+        <Tr verticalAlign="top">
+          <Th> </Th>
+          <Th>
+            <Box display="flex" flexDirection="column" rowGap="space20">
+              Developer
+              <Badge as="span" variant="decorative10">
+                Current plan
+              </Badge>
+            </Box>
+          </Th>
+          <Th>Production</Th>
+          <Th>
+            <Box display="flex" flexDirection="column" rowGap="space20">
+              Business
+              <Badge as="span" variant="success">
+                Recommended
+              </Badge>
+            </Box>
+          </Th>
+          <Th>Personalized</Th>
+        </Tr>
+      </THead>
+      <TBody>
+        <Tr>
+          <Td>Price per month</Td>
+          <Td>Included</Td>
+          <Td>
+            4% of monthly spend <DetailText marginTop="space0">or $250.00 minimum</DetailText>
+          </Td>
+          <Td>
+            6% of monthly spend <DetailText marginTop="space0">or $1500.00 minimum</DetailText>
+          </Td>
+          <Td>
+            8% of monthly spend <DetailText marginTop="space0">or $5000.00 minimum</DetailText>
+          </Td>
+        </Tr>
+        <Tr>
+          <Td>Web support</Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconAvailable" />
+          </Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconAvailable" />
+          </Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconAvailable" />
+          </Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconAvailable" />
+          </Td>
+        </Tr>
+        <Tr>
+          <Td>
+            Guaranteed response times <DetailText marginTop="space0">Business critical</DetailText>
+          </Td>
+          <Td>-</Td>
+          <Td>3 business hours</Td>
+          <Td>1 hour (24x7)</Td>
+          <Td>1 hour (24x7)</Td>
+        </Tr>
+        <Tr>
+          <Td>
+            Guaranteed response times <DetailText marginTop="space0">Degraded service</DetailText>
+          </Td>
+          <Td>-</Td>
+          <Td>6 business hours</Td>
+          <Td>2 business hours</Td>
+          <Td>2 business hours</Td>
+        </Tr>
+        <Tr>
+          <Td>
+            Guaranteed response times <DetailText marginTop="space0">General issues</DetailText>
+          </Td>
+          <Td>-</Td>
+          <Td>9 business hours</Td>
+          <Td>3 business hours</Td>
+          <Td>3 business hours</Td>
+        </Tr>
+        <Tr>
+          <Td>24x7 phone support</Td>
+          <Td>-</Td>
```

Comment:
chore: Sorry, should've caught this earlier. Let's leave the cells blank when the feature doesn't apply to the plan. The content guidelines say to use "n/a", but that's a little too noisy for this context https://paste.twilio.design/components/table#displaying-missing-data